cli/src/main/resources/help/undeploy.txt

```diff
@@ -30,10 +30,13 @@ ARGUMENTS
                     in the standalone mode.
 
  --keep-content   - by default undeploy, besides disabling the deployment, also
-                    removes its content from the repository. The presence of
-                    --keep-content will only disable the deployment w/o removing
-                    its content from the repository. This argument can be used
-                    in both standalone and domain modes.
+                    removes its content from the repository. There is an exception
+                    in domain mode if not all server-groups that reference the
+                    deployment are listed. In this case, the content is kept in
+                    the repository. The presence of --keep-content will only
+                    disable the deployment w/o removing its content from the
+                    repository. This argument can be used in both standalone and
```

Comment:
I think this would be better if this "The --presence of --keep-content..." sentence was dropped and the first line of the help was "only disable the deployment without removing its content from repository. By default...". Currently the text provides quite a bit of background before getting to the key point of what the param does, so that point gets a bit lost.